src/App.js

```diff
@@ -0,0 +1,55 @@
+
+import React,{useState,useEffect} from 'react';
+import Login from "./Components/Login";
+import { BrowserRouter as Router, Routes,Route} from 'react-router-dom';
+import Mainpage from './Components/Main';
+import { auth } from './firebase'; 
+import {login} from "./features/userSlice"
+import {useDispatch} from "react-redux"
+import Form from './Components/Form';
+import Display from './Components/Display';
+
+function App() {
+  
+  const [loggedIn,setLoggedIn]=useState(true);
```

Comment:
Let's follow some standard spacing structure
const [ loggedIn, setLoggedIn ] = useState(true);

Reply:
Always a space before and after "=" and space after a ","